src/js/controllers/chatController.js

```diff
@@ -331,14 +331,21 @@ module.exports = ViewController.extend({
             latestReadTs = parseInt(localStorage.getItem(key) || 0);
         }
         catch (e) {
-        latestReadTs = 0;
+            latestReadTs = 0;
         }
         return latestReadTs;
     },
 
     _setLatestReadTime: function(ts) {
         var key = this._getLatestReadTimeKey();
-        localStorage.setItem(key, ts);
+
+        try {
```

Comment:
Might be worth wrapping localStorage.setItem in a util method so you don't have this duplicated code